src/Account.sol

```diff
@@ -209,45 +209,48 @@ contract Account is IAccount, Auth, OpsReady {
             if (!isOwner()) revert Unauthorized();
 
             if (_command == Command.ACCOUNT_MODIFY_MARGIN) {
+                // Command.ACCOUNT_MODIFY_MARGIN
```

Comment:
Kinda already clear from the if statement right about it no?

Reply:
It is, but half of the command checks do not clearly state "is X == Y" so I added comments for those, and when you're scrolling, it is easier to identify which is which if they're all identified the same way if that makes sense.
More of a preference I guess.

Reply:
Yeah I like it with the comments for consistency - makes it easier to spot a certain point when scrolling.